source/Component/TransformComponent.h

```diff
@@ -1,11 +1,21 @@
 #pragma once
 #include "../Element/Component.h"
+#include "../../dependencies/glm/glm/glm.hpp"
 
 class TransformComponent : public Component
 {
 public:
-	TransformComponent(Entity* anOwner) : Component(anOwner)
+	TransformComponent(Entity* anOwner) : Component(anOwner), myTransform(glm::mat4x4())
```

Comment:
Can't Entity* be made const here?

Reply:
No, we might want to change the owners later inside of the components, so I think its better to leave it non const.

Reply:
If changing ownership of the component is what you are going for, sure, but I can't tell that from the code changes right now. :s